### PR TITLE
feat: use boostagram icon

### LINKED ISSUE #67: Use boostagram icon
See https://podcastfont.com/#boost

### PR DESCRIPTION
**Pull request by theborakompanioni**
Closes #67.

### CODE REVIEW

**Comment by dergigi** [comment]
Thanks for jumping on this @theborakompanioni! 

I think we can put it in front of the "Boostagrams" headline too, and maybe make it a little bit smaller?

File: src/pages/podcast/[shortcode]/[[...slug]].js [hunk 1]
```diff
@@ -47,6 +48,15 @@ const FeaturedBoostagramsHeading = styled.div`
   margin-bottom: 1.5rem;
   color: var(--white);
   cursor: pointer;
+  display: inline-flex;
```
Comment (dnlggr) [inline src/pages/podcast/[shortcode]/[[...slug]].js:51]:
```suggestion
  display: inline-flex;
  align-items: center;
```

File: src/pages/podcast/[shortcode]/[[...slug]].js [hunk 2]
```diff
@@ -47,6 +48,15 @@ const FeaturedBoostagramsHeading = styled.div`
...
+`
+
+const StyledBoostagramBadge = styled(BoostagramBadge)`
+  position: relative;
+  top: -0.2em;
+  width: 1.8rem;
```
Comment (dnlggr) [inline src/pages/podcast/[shortcode]/[[...slug]].js:57]:
```suggestion
  width: 1.5rem;
  height: 1.5rem;
```

**Review by dnlggr — COMMENTED** [review]
Nice! 👍 

The relative positioning doesn't seem to work on Safari for some reason, though:

Screenshot 2022-02-14 at 11 22 13


Let's just center align the flex box. That should work everywhere and also saves us from tinkering with the positioning ourselves. See comments inline.

**Comment by theborakompanioni** [comment]
> I think we can put it in front of the "Boostagrams" headline too, and maybe make it a tiny bit smaller?

I played around a bit. This was what looked best to me, but I do not know if it is what you had it mind.
image

> Let's just center align the flex box. That should work everywhere and also saves us from tinkering with the positioning ourselves. See comments inline.
Thank you. I imagined the icon looked a bit "off" in chrome. Your input in this regards is - as always - quite valuable to me. I should stick more to what is already at hand.

**Review by dnlggr — APPROVED** [review]
LGTM ✅

**Pull request merged**